### PR TITLE
Add verification endpoints

### PR DESCRIPTION
**Pull request by ewood-ac**
Add both the verify DKIM and verify Return-Path endpoint to this code base.

### CODE REVIEW

File: src/Postmark/PostmarkAdminClient.php [hunk 1]
```diff
@@ -345,15 +345,25 @@ function deleteDomain($id) {
...
+
+  /**
+	 * Request that the Postmark API verify Return-Path DNS records associated
+     * with the Domain.
 	 *
-	 * @param  integer $id The ID for the Domain for which we wish to verify the SPF records.
+	 * @param  integer $id The ID of the Domain we wish to verify Return-Path DNS record on.
 	 * @return DynamicResponseModel
 	 */
-	function verifyDomainSPF($id) {
```
Comment (balvig) [inline src/Postmark/PostmarkAdminClient.php:355]: Just to confirm: Are we removing a public feature here as well? 
Do we need to have a deprecation period or other "retirement strategy" in place?
I noticed there's also a mention of this particular method having taken place of _another_ deprecated method:

https://github.com/ActiveCampaign/postmark-php/blob/4ad4180c8ed60740ef1d888cb3ba2b1638080b7c/src/Postmark/PostmarkAdminClient.php#L259

**Review by balvig — COMMENTED** [review]
Nice one picking up maintenance of this lib 👍 

Going to have to lean on your expertise for most of these, but just had one question around deprecation.

File: src/Postmark/PostmarkAdminClient.php (hunk 1 above)
Comment (ewood-ac) [reply src/Postmark/PostmarkAdminClient.php:355]: https://postmarkapp.com/developer/api/domains-api#verify-spf - yes we are removing this. I asked in the backend developer meeting and we should encourage people to use the new methods.

**Comment by wallacio** [comment]
Thanks for adding these two endpoints, v useful. Any chance of making it a release?

**Pull request merged**